### PR TITLE
Clarify the oauth2_auth_token scenario and add the per call credentia…

### PR DESCRIPTION
**Pull request by yang-g**
…ls scenario

### CODE REVIEW

**Comment by yang-g** [comment]
@ejona86

**Comment by yang-g** [comment]
@ejona86 Let me know if we should make the two test cases not send/receive a lot of data. I think it is still easy to change at this point since not a lot of languages have implemented them.

File: doc/interop-test-descriptions.md [hunk 1]
```diff
@@ -396,14 +396,73 @@ Asserts:
...
+actually obtained from the service account credentials via the
+language-specific authorization library.
 
-The difference between this test and the other auth tests is that rather than configuring the test client with ServiceAccountCredentials directly, the test first uses the authorization library to obtain an authorization token.
+The difference between this test and the other auth tests is that rather than
+configuring the test client with ServiceAccountCredentials directly, the test
+first uses the authorization library to obtain an authorization token.
 
 The test
 - uses the flag`--service_account_key_file` with the path to a json key file
```
Comment (ejona86) [inline doc/interop-test-descriptions.md]: Since you are improving formatting, go ahead and add the missing space after "flag"

**Comment by ejona86** [comment]
@yang-g, yeah, let's go ahead and remove the response_* and payload fields in the request message, and similarly remove the assert on response payload body size.

File: doc/interop-test-descriptions.md (hunk 1 above)
Comment (yang-g) [inline doc/interop-test-descriptions.md]: Done.

**Comment by yang-g** [comment]
@ejona86 PTAL

**Pull request merged**